### DISCUSSION TITLE
Welcome to GitHub Discussions!

### DISCUSSION THREAD

**Discussion by davidmyersdev** [davidmyersdev/octo#99] [General]
Hello, friends!

Welcome to [GitHub Discussions](https://docs.github.com/pt/discussions). This is a place to communicate with the community and team behind [octo](https://github.com/voraciousdev/octo). If you have a question about how to use the app, wonder why a specific design decision was made, or want to request a new feature, then just [start a new discussion](https://github.com/voraciousdev/octo/discussions/new)!

**Comment by thomasbnt** [comment]
Pretty cool project! 😀

**Reply by davidmyersdev** [reply]
I appreciate you taking the time to comment! I'm always open to feedback if you have any ideas or questions about something. 😄

**Comment by alDuncanson** [comment]
Neat project!

When I find the time I'd love to try and contribute

**Reply by davidmyersdev** [reply]
Thanks, @alDuncanson! I'm always happy to help out however I can. 😄
